src/status_im/ui/screens/communities/community_overview_redesign.cljs

```diff
@@ -48,121 +50,217 @@
+                     :left             17
+                     :padding          2
+                     :background-color (colors/theme-colors
+                                        colors/white
+                                        colors/neutral-90)}
+            [communities.icon/community-icon-redesign community
+             (min max-image-size (max (- max-image-size (* (+ scroll0 @scroll-height) 3)) min-image-size))]]
+           (when (and (not joined)
+                      (= status :gated))
+             [rn/view (styles/permission-tag-styles)
+              [community-view/permission-tag-container
+               {:locked       locked
+                :status       status
+                :tokens       tokens}]])
+
+           (when joined
+             [rn/view {:position         :absolute
+                       :top              12
+                       :right            12}
+              [status-tags/status-tag {:status {:type :positive} :label (i18n/label :joined)}]])
+           [rn/view  {:margin-top  56}
+            [text/text
+             {:accessibility-label :chat-name-text
+              :number-of-lines     1
+              :ellipsize-mode      :tail
+              :weight              :semi-bold
+              :size                :heading-1} name]]
+
+           [text/text
+            {:accessibility-label :community-description-text
+             :number-of-lines     2
+             :ellipsize-mode      :tail
+             :weight  :regular
+             :size    :paragraph-1
+             :style {:margin-top 8}}
+            description]
+
+           [community-view/community-stats-column :card-view]
+           [community-view/community-tags tags]
+           [preview-user-list]
+           (when (not joined)
+             [button/button
+              {:on-press  #(>evt [:bottom-sheet/show-sheet
+                                  {:content (constantly [request-to-join/request-to-join community])
+                                   :content-height 300}])
+               :override-background-color community-color
+               :style
+               {:width "100%"
+                :margin-top 20
+                :margin-left :auto
+                :margin-right :auto}
+               :before :main-icons2/communities}
+              (i18n/label :join-open-community)])]
+          [channel-list-component channel-heights first-channel-height]]]]])))
 
 (defn overview []
-  (let [community-mock (<sub [:get-screen-params :community-overview]) ;;TODO stop using mock data and only pass community id 
+  (let [community-mock (<sub [:get-screen-params :community-overview]) ; TODO stop using mock data and only pass community id 
         community (<sub [:communities/community (:id community-mock)])]
     [rn/view {:style
-              {:height "100%"}}
-
+              {:position :absolute
+               :height "110%"}}
```

Comment:
set to "100%" as screen is set to absolute for the users phone top bar (where clock, wifi connection etc is displayed) to use the communities background image.

Reply:
How does this work with different platforms .. iOS/Android ?

Reply:
going to look into Android too 👍

Reply:
should be good on android with y updates.

There is further work to abstract this to a generic component and I will improve the handling difference there too so the check for android/ios is only done once 👍